packages/lab/src/navigation-item/NavigationItem.tsx

```diff
@@ -75,23 +75,17 @@ export const NavigationItem = forwardRef<HTMLDivElement, NavigationItemProps>(
     };
 
     return (
-      <div
-        ref={ref}
-        style={style}
-        className={clsx(withBaseName(), className)}
-        {...rest}
-      >
+      <div ref={ref} style={style} {...rest}>
         <ConditionalWrapper
           className={clsx(
-            withBaseName("wrapper"),
+            withBaseName(),
```

Comment:
I find it confusing that the root classname was moved. The style prop and the rest of the props appear on the root element?